src/docs/product/releases/index.mdx

```diff
@@ -23,37 +35,22 @@ We recommend notifying Sentry about a new release before deploying it. But if yo
 
 </Note>
 
```

Comment:
You don't need a repository integration for any of these features, though we recommend [installing one as part of an automated release option](/product/releases/setup/release-automation/) for efficiency.